### DISCUSSION TITLE
Kata 10 - Problemas con el código

### DISCUSSION THREAD

**Discussion by AlbertoVillanuevaE** [LaunchX-InnovaccionVirtual/CursoIntroPython#259] [Q&A]
Hola, tengo un problema al ejecutar el código para detectar el archivo _config.txt_.

Al ejecutar el siguiente código:

> def main():
>     try:
>         configuration = open('config.txt')
>     except FileNotFoundError:
>         print("Couldn't find the config.txt file!")
> 
> 
> if __name__ == '__main__':
>     main()
> 

Debería marcar un error como este:

>$ python config.py
Traceback (most recent call last):
  File "/tmp/config.py", line 9, in <module>
    main()
  File "/tmp/config.py", line 3, in main
    configuration = open('config.txt')
IsADirectoryError: [Errno 21] Is a directory: 'config.txt'

Sin embargo me sigue mostrando el error que dice que no se encuentra el archivo config.txt y así me pasa con cada código que debería marcar una opción distinta, ¿A qué se debe esto?

El directorio lo tengo en la misma carpeta que mi archivo _config.py_

Dejo captura de pantalla de mi código en vs:

![imagen_2022-02-14_210118](https://user-images.githubusercontent.com/99228022/153984573-4e28e606-1b76-4d7e-962c-51b8607b86e4.png)

**Comment by doguedogue** [comment]
Eso pasa en windows, prueba en un bash, terminal linux o wsl y te sale el error que menciona la lección.

https://github.com/LaunchX-InnovaccionVirtual/CursoIntroPython/discussions/214

**Reply by FernandoBarajasValdivia** [reply]
como accedo a mi programa .py desde wsl?

**Reply by doguedogue** [reply]
En WSL se usa python3, checa que lo tengas o lo instalas (python3 -V)

Y la ejecución puede ser: python3 xxx.py
